python/ray/dag/input_node.py

```diff
@@ -177,7 +198,13 @@ class InputAttributeNode(DAGNode):
         >>> ray_dag.execute([2, 3])
     """
 
-    def __init__(self, dag_input_node: InputNode, key: str, accessor_method: str):
+    def __init__(
+        self,
+        dag_input_node: InputNode,
+        key: str,
+        accessor_method: str,
+        input_type: str = None,
+    ):
```

Comment:
Could we add a docstring documenting what these attributes are? Alternatively, could we add a comment explaining what `input_type` does? It's not clear at first glance that it's tied directly (and only) to the Gradio integration.

Reply:
I don't fully know what `accessor_method` is used for, so for now I've added a comment explaining `input_type`.

Reply:
Sounds good!